articles/azure/azs-how-add-disks.md

```diff
@@ -0,0 +1,245 @@
+---
+title: How to add disks on Azure Stack
+description: How to add disks on Azure Stack
+services: azure-stack
+author: David Woffendin
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2: Disk Management
+toc_sub3:
+toc_sub4:
+toc_title: How to add disks on Azure Stack
+toc_fullpath: Users/How To/Disk Management/azs-how-add-disks.md
+toc_mdlink: azs-how-add-disks.md
+---
+# Adding disks on Azure Stack
+
+## Overview
+
+In Azure Stack there are two types of disks, **Managed** and **Unmanaged** both these disks can be used on VMs as both OS and data disks.
+
+### Managed disks
+
+Managed disks simplify disk management for Azure IaaS VMs by managing the storage accounts associated with the VM disks. You only have to specify the size of disk you need, and Azure Stack creates and manages the disk for you.
+
+### Unmanaged disks
+
+Unmanaged disks require that you create a storage account to store the disks. The disks you create are referred to as VM disks and are stored in containers in the storage account.
+
+## Adding a new disk to a VM using the portal
+
+1. In the portal select **Virtual machines**.
+
+2. Select the VM you want to add disks to.
+
+3. Navigate to **Disks** > **+Add data disk**.
+
+    ![Azure Stack disk creation disks blade](images/azs-browser-disk-creation.png)
+
+4. Edit the options for your new disk.
+
+    - **Name**: This is what the disk will be called.
+
+    - **Source type**: Leave this as **New(empty disk)** as we are creating a new disk.
+
+    - **Account type**: This is where you choose between SSD and HDD.
+
+    - **Size**: This is where you specify the size of the new disk.
+
+    - **Storage container**: This is where you specify the storage container for the disk. Find the storage account where your VMs OS disk is located and then create a new a new storage container and then select this.
```

Comment:
This is an example of where you do need an apostrophe in VM's (because you're talking about the OS disk that belongs to the VM).

Suggest a slight change of wording:

This is where you specify the storage container for the disk. Find the storage account where your VM's OS disk is located, create a new storage container then select the new container.

Reply:
Ill try to keep this in mind in future, and done